### ISSUE TITLE
Patch for handling multiple pages on one page without style overlapping

### ISSUE THREAD

**Issue by HelgaOR** [coolwanglu/pdf2htmlEX#590]
Hello,

First of all, thanks for incredibly working tool! It really has no match in what it does.

We were using this tool to convert some books (separate pages) and then publish it on some site. We had no control of that site, but needed it to work correctly. So there was a search engine, that displayed all html pages that match, on one page, without iframes or whatever, with highlight for found words.

To make this work, we applied a small patch for this code, replacing static class names by something that won't overlap, and even if it will, it won't matter.

Basically, for one-double-value classes we formed class names based on the actual value, something like:

    long long id = (long long)new_value*100;

instead of

    long long id = value_map.size();

(please see [this](https://github.com/Shurf/pdf2htmlEX/commit/cf9aba96a1ef6c7247949fe3df0c7069ee94e1e7) commit for details)

For some classes, though, we couldn't implement such method, so for our needs we just appended current time so that they would be always different.

I wanted to let you know of this solution in case you choose to add it to your project.

**Comment by coolwanglu** [comment]
There was a idea about support custom prefix for all the class names, which can be specified via commandline. But it has not been implemented yet unfortunately.
